### PR TITLE
Implement Toyzero Cropping/Tiling Scripts

### PR DESCRIPTION
**Pull request by usert5432**
Hi everyone,

I have implemented the scripts to center crop and crop tile the toyzero dataset, according to the specifications that we have discussed. Before I run them to create the final toyzero crops, could I ask you please to review these scripts?

Inviting: @HaiwangYu, @pphuangyi.

### CODE REVIEW

File: scripts/toyzero/center_crop [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    os.makedirs(outdir_fake, exist_ok = True)
+
+    progbar = tqdm.tqdm(
+        desc = 'Cropping', total = len(images), dynamic_ncols = True
+    )
+
+    worker = CroppingWorker(root, shape, outdir_real, outdir_fake)
+
+    with multiprocessing.Pool() as pool:
+        for _ in pool.imap_unordered(worker, enumerate(images)):
```
Comment (YHRen) [inline scripts/toyzero/center_crop:206]: cool. didn't know tqdm works with multiprocesssing like this. 👍

**Review by YHRen — APPROVED** [review]
@usert5432  everything looks good to me. Just to double check, in this setting, the training events of the two domains are exactly the same, since we are using the same indices. The "unpaired-ness" is done by shuffle them differently in the mini-batch. correct? (so we need to make sure data-1 loader and data-2 loader have different random seed.)

**Review by brettviren — APPROVED** [review]
Looks good.  One random code comment: I've come to enjoy using the standard `pathlib` instead of `os.path` for mucking about with file paths.  Both are fine.

**Comment by YHRen** [comment]
> Looks good. One random code comment: I've come to enjoy using the standard `pathlib` instead of `os.path` for mucking about with file paths. Both are fine.

on the same boat. ;)

**Comment by usert5432** [comment]
Thanks @brettviren and @YHRen for the review.

> Just to double check, in this setting, the training events of the two domains are exactly the same, since we are
> using the same indices. The "unpaired-ness" is done by shuffle them differently in the mini-batch. correct?

@YHRen, yes.

> (so we need to make sure data-1 loader and data-2 loader have different random seed.)

That is a great point. I have assumed that the two data loaders will have different seeds by default, but I have never tested that. I guess we need to run some tests for `toygan`.

**Pull request closed without merging**